src/21-pubsub/index.md

```diff
@@ -0,0 +1,105 @@
+---
+# this is the final blog post's id (used in the directory)
+# this id is part of the url, and should only contain:
+#   letters, numbers, dashes.
+id: 21-pubsub
+
+breadcrumbs:
+  - {name: "Take a look at pubsub on ipfs!", link: "./" }
+
+# the date here should be set to the final publication date,
+# on the day it is published.
+date: 2016-11-09
+
+# this is the Title
+title: Take a look at pubsub on ipfs!
+
+# this is the name of the main author(s)
+author: Jeromy Johnson
+
+# technical details required for the software, don't change these.
+baseurl: ..
+template: tmpl/layouts/post.html
+collection: posts
+---
+
+We are very excited to announce that we have recently merged a simple,
+experimental pubsub implementation.  While the implementation is experimental,
```

Comment:
@jbenet also, i'm really not sure what you want for the 'roadmap' bit, mind filling that part in?